### ISSUE TITLE
584109C2 - Perfect Dark Beta 102

### ISSUE THREAD

**Issue by ProgrammingHider** [xenia-project/game-compatibility#1365]
[Marketplace](http://marketplace.xbox.com/en-AU/Product/Perfect-Dark/66acd000-77fe-1000-9115-d802584109c2)

Tested on https://github.com/xenia-project/xenia/commit/bc2eaf9b

# Issues:
Only Minor Graphical Glitches and a Bit of Lag

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3438506/xenia.zip)

# Screenshot(s):
![xenia_XVjUGQ14HB](https://user-images.githubusercontent.com/38049865/61998605-d2464a80-b0b2-11e9-9c96-293b0d1dea64.jpg)
![xenia_NfKbxeo5mP](https://user-images.githubusercontent.com/38049865/61998606-d5413b00-b0b2-11e9-94fd-d9e93a47708f.png)
![xenia_htPFVcg8gW](https://user-images.githubusercontent.com/38049865/61998608-d83c2b80-b0b2-11e9-80f0-a25244e79b31.jpg)
![xenia_pytMYyGnjU](https://user-images.githubusercontent.com/38049865/61998614-dc684900-b0b2-11e9-9bf1-d50fd80fb5e6.jpg)

# Labels:
(state-playable)
(gpu-corrupt-drawing)

**Comment by ProgrammingHider** [comment]
Update:
Running the D3D12 Version of Xenia fixes the Texture Noise, but the texture around the fans is stil broken

![xenia_t8KeaBStjJ](https://user-images.githubusercontent.com/38049865/61999615-c748e580-b0c5-11e9-9c13-e07705fec4e3.jpg)
![xenia_gvYsFwmK5g](https://user-images.githubusercontent.com/38049865/61999616-cadc6c80-b0c5-11e9-91ac-5a187680267b.jpg)

**Comment by katastic** [comment]
Which version? I just downloaded master (39c3f72c) and it still has the texture noise.

![image](https://user-images.githubusercontent.com/11449593/62676857-cfc8d800-b971-11e9-9aa1-39afa1dd99d1.png)


first line of xenia.log:
"i> 000031F8 Build: detached / 39c3f72c41fa08a65e93bbe1fd2f408099dd9db7 on Aug  8 2019"

**Comment by doriveramirez** [comment]
Me too, I downloaded master (af93986c) and it still has the texture noise.

**Comment by ProgrammingHider** [comment]
I used bc2eaf9 and it didn't have Texture Noise

**Comment by illusion98** [comment]
If you are using an AMD GPU there won't be much noise

**Comment by Margen67** [comment]
@ProgrammingHider Was this ever released?

**Comment by tjd2** [comment]
Didn't you mean build 0033? I had build 0102, but it was the same as retail.
Build 0033 has got debug menu in settings.

**Comment by DoctorCharles** [comment]
Someone get to work the combat simulator?

**Comment by pwnedbygary** [comment]
> Someone get to work the combat simulator?

Yes, the combat sim is working in the latest build I tested today. Not sure when it was made compatible, but I was able to start a game by myself.

**Comment by manuelrojaspiaggio** [comment]
So, I do not find the bc2eaf9. Where may I find it? Or is any other way to fix it yet? Thank you, people!

**Comment by pwnedbygary** [comment]
> So, I do not find the bc2eaf9. Where may I find it? Or is any other way to fix it yet? Thank you, people!

You can find various things in the megathread here: https://r-roms.gitlab.io/megathread/

**Comment by madmodder123** [comment]
> So, I do not find the bc2eaf9. Where may I find it? Or is any other way to fix it yet? Thank you, people!

There is currently no way to fix the texture noise if using an NVIDIA GPU

**Comment by TDOdev91** [comment]
I want to bring to everyones attention coming on this thread that the shader bug for Nvidia GPU users is indeed now fixed. The direct cause was located and has been patched as of today for Xenia Canary and will be available shortly. You can acquire the patch now by joining the Xenia discord. Below are GitHub references were this has been solved.
https://github.com/xenia-project/game-compatibility/issues/316
https://github.com/xenia-canary/xenia-canary/pull/268
